src/components/ListItem.jsx

```diff
@@ -15,22 +17,48 @@ export function ListItem({
 	dateLastPurchased,
 	dateNextPurchased,
 	totalPurchases,
+	refTime,
+	daysToNext,
 }) {
 	const DAYINMS = 86400000;
 	// sync up checked or not checked data from the database to the page upon page refresh
 	const [isPurchased, setIsPurchased] = useState(isChecked);
 
+	const urgency = [
+		['overdue', 'Overdue'],
+		['soon', 'Soon'],
+		['kind-of-soon', 'Kind Of Soon'],
+		['not-soon', 'Not Soon'],
+		['inactive', 'Inactive'],
+	];
```

Comment:
In larger codebases, this code can be a little difficult to read and understand since the values need to be called using array indices (`urgency[0]` and `urgency[1]`, etc). This is not necessary, but it's usually better to use an enum object here, like:

```javascript
const urgency = {
  OVERDUE: 'Overdue',
  SOON: 'Soon',
  ...
}
```

That way, when you use it, it can be called like `urgency.OVERDUE` instead of `urgency[0]`.

Reply:
Makes sense! For the class name, we can also just process the same string value.

Reply:
Pushed an update - `urgency` is now an object, but I kept the arrays in favor of avoiding additional string processing for className purposes for now.